### ISSUE TITLE
ERROR SAW-A40704

### ISSUE THREAD

**Issue by JungTaeKim103** [STOmics/SAW#86]
Hi, I am jungtae kim in Kaist, south korea. 

Today, I got the error code, SAW-A40704. 
The SAW user manual A8.1 mentioned the invalid data or data exception. 
In detail, 
Error data. Please check the imageTools ipr2img input IPR file content. This error may arise because the CZI file in TAR.GZ is invalid, or the image has not either automatically or manually registered with the expression matrix. The second circumstance can be confirmed from IPR by checking whether the StereoResepSwitch/register is TRUE (has not performed automatic registration) or the ManualState/register is FALSE (has not performed manual registration). The third possible reason is the StereoResepSwitch/tissueseg 
is TRUE, therefore cannot output cell segmentation result. 

Since I used Leica microscopy, I just have the TIF file. So I think "the CZI file in TAR.GZ is invalid" was not my case.
I tried to solve the second circumstance and the third possible reason.  by checking whether the StereoResepSwitch/register is TRUE, 
not performed automatic registration) or the ManualState/register is FALSE, or StereoResepSwitch/tissueseg is TRUE
However, I was not able to open the .ipr file. 

I am really not good at informatics. Can anybody help me?

**Comment by JungTaeKim103** [comment]
I used stereoPipeline_v7.0.sh
```
bash stereoPipeline_v7.0.sh \
    -sif /home/users/Minhos/SAW_7.0.sif \
    -splitCount 1 \
    -maskFile /home/users/R/Stereoseq/*******barcodeToPos.h5 \
    -fq1 /home/users/R/Stereoseq/Mouse-Brain-Control.R1.fq.gz \
    -fq2 /home/users/R/Stereoseq/Mouse-Brain-Control.R2.fq.gz \
    -speciesName Mouse \
    -tissueType Brain \
    -refIndex /home/users/R/Stereoseq/reference/GRCm39/genome/STAR_SJ100/ \
    -annotationFile /home/users/R/Stereoseq/reference/GRCm39/genes/Mus_musculus.GRCm39.104.gtf \
    -rRNAremove N \
    -threads 36 \
    -outDir /home/users/R/Stereoseq/results/ \
    -imageRecordFile /home/users/R/Stereoseq/*******_SC_20240116_133733_3.0.2.ipr \
    -imageCompressedFile /home/users/R/Stereoseq/*******_SC_20240116_133733_3.0.2.tar.gz \
    -doCellBin Y 
```

**Comment by Jinghao-Chen** [comment]
Hi, could you show us the log of the script you have run and the log files in the 03.register directory? Maybe it's because your register step was not executed correctly. For ipr files. you could use h5py in python or hdfview on Windows to view them.

**Comment by JungTaeKim103** [comment]
[20240116-18-34-38.log](https://github.com/STOmics/SAW/files/13956848/20240116-18-34-38.log)
[20240116-19-31-53.log](https://github.com/STOmics/SAW/files/13957913/20240116-19-31-53.log)
[20240116-23-40-56.log](https://github.com/STOmics/SAW/files/13957914/20240116-23-40-56.log)

It is the log of the 03. register.

**Comment by JungTaeKim103** [comment]
![image](https://github.com/STOmics/SAW/assets/156797080/01ae401c-d797-4b57-93a8-6427451dc741)
![image](https://github.com/STOmics/SAW/assets/156797080/880b79ad-c982-47a7-8dcc-1fdaa4b122ae)
I was not able to get the information about StereoResepSwithch/register, ManualState/register, or StereoResepSwitch/tissueseg to confirm whether it is TURE or FALSE

**Comment by Jinghao-Chen** [comment]
> I was not able to get the information about StereoResepSwithch/register, ManualState/register, or StereoResepSwitch/tissueseg to confirm whether it is TURE or FALSE

Because it is the ipr file before running SAW. 
Could you confirm how many ipr files there are in 03.register?

**Comment by JungTaeKim103** [comment]
I appreciate you for your prompt reply. 
I think it is working well now. 

I fixed two things. 
1. I deleted all of ipr files in 03.register
2. I suddenly recognized when I wrote the microscope information in stereomap, the information was the default value. so, I modified the information to fit with our microscope.

I am not sure which one was the problem. But, my first sample is working well now. 

Thank you so much.
